src/pages/HomePage/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from 'react';
+import { MdSearch } from 'react-icons/md';
+
+import {
+	Container,
+	SearchBar,
+	Empty,
+	Illustration,
+	Title,
+	Subtitle,
+	Results,
+} from './styles';
+import Logo from '../../components/Logo';
+import MovieCard from './MovieCard';
+
+import { searchMovies } from '../../services/api';
+
+export default function HomePage() {
+	const [results, setResults] = useState([]);
+	const [value, setValue] = useState('');
+
+	useEffect(() => {
+		if (value === '') {
+			setResults([]);
+			return;
+		}
+
+		searchMovies(value).then(({ data }) =>
+			setResults(data.results.filter((item) => item.poster_path))
+		);
+	}, [value]);
```

Comment:
**Feature suggestion**: What about put the query search as a query string on URL? 

My navigation flow was like this:
1. I searched for `007` for example;
2. Clicked on the first movie, then I entered on the page movie;
3. Then I went back to the previous page (search page), and I noticed that my search wasn't there anymore; 
4. The user experience would be better to persist the last query search.